src/components/Post/NewCommentScreen.js

```diff
@@ -34,7 +34,7 @@ export default function NewCommentScreen({ navigation }) {
     };
     Firestore().collection('comments')
       .doc()
-      .add(commentRecord)
+      .set(commentRecord)
```

Comment:
if you want to use `.add(commentRecord)` here, i think you would replace `.doc().set(commentRecord)` with `.add(commentRecord)`.